loopback/index.d.ts

```diff
@@ -0,0 +1,2585 @@
+// Type definitions for loopback 2.3.4
+// Project: https://github.com/strongloop/loopback
+// Definitions by: Andres D Jimenez <https://github.comhttps://github.com/kattsushi/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+/************************************************
+*                                               *
+*               LoopBack v2.x API               *
+*                                               *
+************************************************/
+
+/**
+* The `App` object represents a Loopback application.
+*
+* The App object extends [Express](expressjs.com/api.html#express) and
+* supports Express middleware. See
+* [Express documentation](expressjs.com) for details.
+*
+* ```js
+* var loopback = require('loopback');
+* var app = loopback();
+*
+* app.get('/', function(req, res){
+*   res.send('hello world');
+* });
+*
+* app.listen(3000);
+* ```
+*
+* @class LoopBackApplication
+* @header var app = loopback()
+*/
+
+declare class LoopBackApplication {
```

Comment:
`declare class` in this context declares a global variable. Is that really what you want? Or is it an export of some module?